easybuild/easyconfigs/i/Infernal/Infernal-1.1.4-foss-2020b.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# Updated:: Denis Kristak (INUITS)
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'Infernal'
+version = "1.1.4"
+
+homepage = 'http://eddylab.org/infernal/'
+description = """Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases
+ for RNA structure and sequence similarities."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://eddylab.org/%(namelower)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f9493c7dee9fbf25f6405706818883d24b9f5e455121a0662c96c8f0307f95fc']
+
+sanity_check_paths = {
+    'files': ['bin/cm%s' % x for x in ['align', 'build', 'calibrate', 'convert', 'emit',
```

Comment:
Do any of these support `--help` or `--version`, or similar?
If so, please add a sanity check command for some (or all) of them.

If you do all, introduce a local variable `local_bins = ['bin/cm%s' % x for x in ['align', ...]]`